src/js/popup/popup.js

```diff
@@ -1,802 +1,1350 @@
+                readNewsItemCount: 0,
+              });
+            }
+
+            // FIXME: The total news item count may differ than what is displayed to the user
+            // Example: Three items total but user doesn't have waffle for one news item. 
+            // Regardless - update the unreadNews count to match whatever is in state
+            await browser.storage.local.set({
+              unreadNewsItemsCount: state.newsItemsCount,
+            });
+
+            const { readNewsItemCount } = await browser.storage.local.get(
+              "readNewsItemCount"
+            );
+
+            const { unreadNewsItemsCount } = await browser.storage.local.get(
+              "unreadNewsItemsCount"
+            );
+
+            // Set unread count
+            const newsItemCountNotification = document.querySelector(
+              ".fx-relay-menu-dashboard-link[data-panel-id='news'] .news-count"
+            );
+            
+            const unreadCount = unreadNewsItemsCount - readNewsItemCount;
+
+            // Show count is it exists
+            if (unreadCount > 0) {
+              newsItemCountNotification.textContent = unreadCount.toString();
+              newsItemCountNotification.classList.remove("is-hidden");
+            }
+            
+          },
+          updateNewsItemCountNotification: async (markAllUnread = false) => {
+            if (markAllUnread) {
+              await browser.storage.local.set({
+                readNewsItemCount: state.newsItemsCount,
+              });
+
+              const newsItemCountNotification = document.querySelector(
+                ".fx-relay-menu-dashboard-link[data-panel-id='news'] .news-count"
+              );
+
+              newsItemCountNotification.classList.add("is-hidden");
+
+            }
+          }
+        },
+      },
+      settings: {
+        init: () => {
+          popup.utilities.enableInputIconDisabling();
+
+          // Function is imported from data-opt-out-toggle.js
+          enableDataOptOut();
+
+          const reportWebcompatIssueLink = document.getElementById("popupSettingsReportIssue");
+            
+          if (state.loggedIn) {
+            reportWebcompatIssueLink.classList.remove("is-hidden");
+            reportWebcompatIssueLink.addEventListener("click", (e) => {
+                e.preventDefault();
+                popup.panel.update("webcompat");
+              }, false);
+          } else {
+            reportWebcompatIssueLink.classList.add("is-hidden");
+          }
+        }
+      },
+      stats: {
+        init: async () => {
+          // Get Global Mask Stats data
+          const { aliasesUsedVal } = await browser.storage.local.get(
+            "aliasesUsedVal"
+          );
+          const { emailsForwardedVal } = await browser.storage.local.get(
+            "emailsForwardedVal"
+          );
+          const { emailsBlockedVal } = await browser.storage.local.get(
+            "emailsBlockedVal"
```

Comment:
Question: We are not destructuring here right? Do we still need the curly braces?

Reply:
Nice! Good to know. I saw the `.get('aliasesUsedVal')` and figured it would return the value for just that - thanks!